src/docs-components/IconDocs.tsx

```diff
@@ -0,0 +1,78 @@
+import { icons, IconName } from '@virtuslab/tetrisly-icons';
+import { FC } from 'react';
+
+import { SectionHeader } from './common/SectionHeader';
+
+import { Icon } from '@/components/Icon';
+import { tet } from '@/tetrisly';
+import { MarginProps } from '@/types';
+
+const iconsNames = Object.keys(icons) as IconName[];
+
+const icons20 = iconsNames.filter((iconName) => iconName.startsWith('20-'));
+const icons16 = iconsNames.filter((iconName) => iconName.startsWith('16-'));
+
+const IconsBoard: FC<{ heading: string; items: IconName[] } & MarginProps> = ({
+  heading,
+  items,
+  ...rest
+}) => (
+  <tet.div {...rest}>
+    <SectionHeader variant="H1" as="h3" mb="$dimension-300">
+      {heading}
+    </SectionHeader>
+    <tet.div
+      display="grid"
+      gridTemplateColumns="repeat(2, 1fr)"
+      ringInset
+      ring="$border-width-small"
+      ringColor="$color-border-default"
+      borderRadius="$border-radius-medium"
+      overflow="hidden"
+    >
+      {items.map((iconName) => (
+        <tet.div
+          display="flex"
+          alignItems="center"
+          py="$dimension-200"
+          px="$dimension-300"
+          borderBottom
+          borderRight
+          borderColor="$color-border-default"
+          key={iconName}
+        >
+          <tet.div
+            display="flex"
+            alignItems="center"
+            justifyContent="center"
+            w="$size-medium"
+            h="$size-medium"
+            borderRadius="$border-radius-medium"
+            backgroundColor="$color-background-neutral-subtle"
+          >
+            <Icon color="$color-content-primary" name={iconName as IconName} />
```

Comment:
That casting is unnecessary
```suggestion
            <Icon color="$color-content-primary" name={iconName} />
```